src/Raven.Studio/typescript/viewmodels/database/status/ongoingTasksStats.ts

```diff
@@ -1253,6 +1280,8 @@ class ongoingTasksStats extends viewModelBase {
             const batchPerfLength = batchPerformance.length;
             
             let lastErrorPosition = -1000;
+
+            const workersArray = [...this.subscriptionToWorkers.get(subscriptionItem.TaskName)];
```

Comment:
do we need shallow copy here? 
Let's create map from task name to index - it will be faster

Reply:
then inside draw connection for (below) we will have faster lookup

Reply:
done